src/server/pom.xml

```diff
@@ -769,4 +794,29 @@
             </plugin>
         </plugins>
     </reporting>
+
+    <repositories>
+        <repository>
+            <id>dependency-snapshots-repo</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases>
+            <enabled>false</enabled>
+            </releases>
+            <snapshots>
+            <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>plugin-snapshots-repo</id>
```

Comment:
Do we really need a plugin snapshot repository? I don't see any plugin sporting a snapshot version.

Reply:
We don't indeed. Removed ✅